Website/src/rendering/src/components/Checkout/LineItemCard.tsx

```diff
@@ -0,0 +1,87 @@
+import { useCallback, useState } from 'react';
+import { DLineItem } from 'src/models/ordercloud/DLineItem';
+import useOcProduct from '../../hooks/useOcProduct';
+import { patchLineItem, removeLineItem } from '../../redux/ocCurrentCart';
+import QuantityInput from './QuantityInput';
+import GiftCheckboxLineItem from './GiftCheckboxLineItem';
+import { useAppDispatch } from 'src/redux/store';
+
+type LineItemCardProps = {
+  lineItem: DLineItem;
+  editable?: boolean;
+};
+
+const LineItemCard = (props: LineItemCardProps): JSX.Element => {
+  const dispatch = useAppDispatch();
+  const [loading, setLoading] = useState(false);
+
+  const product = useOcProduct(props.lineItem.ProductID);
+
+  const getProductName = () => {
+    const li = props.lineItem;
```

Comment:
We try to avoid short variable names that are not indicating their role like `s` and `li` in our projects. Could this one please be changed to `lineItem`?